test/integration/print_config.cc

```diff
@@ -0,0 +1,147 @@
+      [&](const std::string &_file)
+      {
+        return sdf::filesystem::append(modelPath, _file);
+      });
+
+  const std::string includeStr =
+R"(<include>
+  <uri>box</uri>
+  <name>test_box</name>
+  <pose>1 2 3 0 0 0</pose>
+  <placement_frame>link</placement_frame>
+  <plugin name='test_plugin' filename='test_plugin_file'/>
+</include>
+)";
+
+  const std::string modelWithIncludeStr =
+R"(<model name='test2'>
+  <include>
+    <uri>test_model</uri>
+    <name>test_model</name>
+    <pose>1 2 3 0 0 0</pose>
+    <placement_frame>link</placement_frame>
+    <plugin name='test_plugin' filename='test_plugin_file'/>
+  </include>
+</model>
+)";
+
+  const std::string sdfStr =
+    "<sdf version='1.9'>"
+    "  <world name='default'>"
+    + includeStr +
+    modelWithIncludeStr +
+    "  </world>"
+    "</sdf>";
+
+  sdf::Root root;
+  sdf::Errors errors = root.LoadSdfString(sdfStr);
+  EXPECT_TRUE(errors.empty()) << errors;
+
+  auto *world = root.WorldByIndex(0);
+  ASSERT_NE(world, nullptr);
+  auto *includedModel = world->ModelByIndex(0);
+  ASSERT_NE(includedModel, nullptr);
+  auto *modelWithInclude = world->ModelByIndex(1);
+  ASSERT_NE(modelWithInclude, nullptr);
+
+  const std::string expandedIncludeStr =
+R"(<model name='test_box' placement_frame='link'>
+  <pose>1 2 3 0 0 0</pose>
+  <link name='link'>
+    <collision name='collision'>
+      <geometry>
+        <box>
+          <size>1 1 1</size>
+        </box>
+      </geometry>
+    </collision>
+    <visual name='visual'>
+      <geometry>
+        <box>
+          <size>1 1 1</size>
+        </box>
+      </geometry>
+    </visual>
+  </link>
+  <plugin name='test_plugin' filename='test_plugin_file'/>
+</model>
+)";
+
+  const std::string expandedModelWithIncludeStr =
+R"(<model name='test2'>
+  <model name='test_model' placement_frame='link'>
+    <link name='link'>
+      <collision name='mesh_col'>
+        <geometry>
+          <mesh>
+            <uri>meshes/mesh.dae</uri>
+            <submesh>
+              <name>my_submesh</name>
+              <center>true</center>
+            </submesh>
+            <scale>0.1 0.2 0.3</scale>
+          </mesh>
+        </geometry>
+      </collision>
+      <visual name='mesh_vis'>
+        <geometry>
+          <mesh>
+            <uri>meshes/mesh.dae</uri>
+            <submesh>
+              <name>another_submesh</name>
+              <center>false</center>
+            </submesh>
+            <scale>1.2 2.3 3.4</scale>
+          </mesh>
+        </geometry>
+      </visual>
+    </link>
+    <pose>1 2 3 0 0 0</pose>
+    <plugin name='test_plugin' filename='test_plugin_file'/>
+  </model>
+</model>
+)";
+
+  sdf::PrintConfig config;
+  // by default, included model should be expanded
+  EXPECT_EQ(includedModel->Element()->ToString("", config), expandedIncludeStr);
+  EXPECT_EQ(modelWithInclude->Element()->ToString("", config),
+            expandedModelWithIncludeStr);
+
+  config.SetPreserveIncludes(true);
```

Comment:
Is it possible to also test `//include/merge=true`?

Reply:
Per our discussion yesterday with #769 , added a negative test d9ac7e021fdef084154c942873b9434b2e78642f